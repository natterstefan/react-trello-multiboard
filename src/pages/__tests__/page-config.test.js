import React from 'react'
import { shallow, mount } from 'enzyme'
import { HashRouter as Router } from 'react-router-dom'
import SyntaxHighlighter from 'react-syntax-highlighter'

import ConfigPage from '../page-config'

describe('Page/Config', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<ConfigPage />)
    expect(wrapper).toMatchSnapshot()
  })

  test('should render with SyntaxHighlighter as a child of the page', () => {
    const wrapper = mount(
      <Router>
        <ConfigPage />
      </Router>,
    )
    expect(wrapper.find(ConfigPage)).toHaveLength(1)
    expect(wrapper.find(SyntaxHighlighter)).toHaveLength(1)
  })
})
