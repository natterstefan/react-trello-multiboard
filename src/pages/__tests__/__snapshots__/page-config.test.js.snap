// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page/Config should render without throwing an error 1`] = `
<ConfigPage
  classes={
    Object {
      "root": "ConfigPage-root-1",
    }
  }
/>
`;
